src/AppMap.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import Leaflet from 'leaflet';
+import 'leaflet/dist/leaflet.css';
+import {
+  Map,
+  Polyline,
+  TileLayer,
+  Marker,
+  Popup
+} from 'react-leaflet';
+import '../map.css';
+import Spinner from 'react-bootstrap/Spinner';
+import { NoPaths, getData } from '../Shared';
+import Tab from 'react-bootstrap/Tab';
+import Tabs from 'react-bootstrap/Tabs';
+
+// pull images from cdn instead of storing locally
+Leaflet.Icon.Default.imagePath = '//cdnjs.cloudflare.com/ajax/libs/leaflet/1.3.4/images/';
+
+export default class AppMap extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {markerData: [], isLoading: true};
+	}
+
+	async componentWillMount() {
+		// urls for the direction jsons
+		var marker_urls;
+		const names = await getData('https://github.com/SASE-Labs-2021/gopher-a-bite/api/restNames.json');
```

Comment:
Please test the code before opening a pull request. We don't want to merge code that hasn't been tested.